cardano-api/ChangeLog.md

```diff
@@ -35,7 +35,7 @@ None
 ## 1.21.2 -- September 2020
 
 - Add a Ed25519-BIP32 instance of the new crypto classes (#1933, #1952)
-- Adjust what is exposed via Cardano.API.{Byron,Shelley} (#1932)
+- Adjust what is exposed via Cardano.Api.{Byron,Shelley} (#1932)
```

Comment:
We shouldn't change the changelogs, right?

Reply:
I don't think it matters too much in this case, it might even be helpful.